app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetController.kt

```diff
@@ -493,6 +495,7 @@ class BloodPressureEntrySheetController @Inject constructor(
               .flatMap { bpSaved ->
                 appointmentRepository
                     .markAppointmentsCreatedBeforeTodayAsVisited(newBp.patientUuid)
+                    .andThen(patientRepository.updatePatientRecordedAt(newBp.patientUuid, toUtcInstant(newBp.date)))
```

Comment:
Should we move this `toUtcInstant` lambda to `TimeFunctions.kt`? We have a lot of these useful utility methods related to Time that we end up rewriting a lot.